main.go

```diff
@@ -184,6 +185,9 @@ func main() {
 		Logger:                 setupLog,
 	}
 
+	mgrOptions.Client.WarningHandler.SuppressWarnings = true
+	rest.SetDefaultWarningHandler(&rest.NoWarnings{})
```

Comment:
Could those warnings be logged with the debug level by creating a custom WarningHandler?

Reply:
They could, and maybe in the future there are other warnings that may be all we get. let me change that to debug logging.